### PR TITLE
MGMT-3701 Update AI to use latest release image that support SNO

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: internal/ignition/ignition.go [hunk 1]
```diff
@@ -155,11 +155,16 @@ create_template_file
 correlate_int_mac
 `
 
+const (
+	masterIgn = "master.ign"
```
Comment (eranco74) [inline internal/ignition/ignition.go:159]: bootstrap.ign?

File: internal/ignition/ignition.go [hunk 2]
```diff
@@ -383,6 +380,30 @@ func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte)
...
+
+	// In case of single node rename bootstrap Ignition file
+	err = os.Rename(filepath.Join(g.workDir, "bootstrap-in-place-for-live-iso.ign"), filepath.Join(g.workDir, "bootstrap.ign"))
+	if err != nil {
+		return errors.Wrapf(err, "Failed to rename bootstrap-in-place-for-live-iso.ign")
+	}
+
+	config := config_31_types.Config{Ignition: config_31_types.Ignition{Version: config_31_types.MaxVersion.String()}}
+
+	for _, file := range []string{masterIgn, workerIgn} {
```
Comment (eranco74) [inline internal/ignition/ignition.go:397]: No need for the worker 
No need for the loop

Comment (tsorya) [reply internal/ignition/ignition.go:397]: need worker. Without worker code will fail. Assisted expects it.

File: internal/ignition/ignition.go (hunk 1 above)
Comment (tsorya) [reply internal/ignition/ignition.go:159]: next time ? :) 
wanted to consts only those i actually touch :)

**Pull request merged**